core/src/main/resources/lib/hudson/progressive-text.css

```diff
@@ -0,0 +1,12 @@
+div.progressiveText-block:not(:first-child) {
+  animation: progressiveText_highlight 1s;
+}
+
+@keyframes progressiveText_highlight {
+  0% {
+    background: rgba(0, 0, 100, 0.2);
+  }
+  100% {
+    background: rgba(0, 0, 100, 0);
+  }
+}
```

Comment:
Looks good to me! Might be worth using a pseudo element like so: 

```
div.progressiveText-block:not(:first-child) {
  position: relative;

  &::before {
    content: "";
    position: absolute;
    inset: 0 -1rem;
    animation: progressiveText_highlight 1.5s ease;
    animation-fill-mode: both;
    background: var(--primary);
    opacity: 0.3;
    pointer-events: none;
  }
}

@keyframes progressiveText_highlight {
  to {
    opacity: 0;
  }
}
```

so we can use the --primary colour variable with opacity. This also allows us to have the flash background be full width.

Reply:
Could you provide that as CSS so I can simply insert it into the file and see what happens? I tried to "translate" but it doesn't actually disappear, it's just a colored background.